networkparams_test.go

```diff
@@ -49,7 +51,7 @@ func checkPowLimitsAreConsistent(t *testing.T, params chaincfg.Params) {
 //
 // This test ensures these blocks will respect the network PoW limit.
 func checkGenesisBlockRespectsNetworkPowLimit(
-	t *testing.T, params chaincfg.Params) {
+	t *testing.T, params *chaincfg.Params) {
```

Comment:
Please put this on the same line as the function def so it is consistent with the rest.  Function definitions split on multiple lines causes some undesirable behavior with various tooling.